pkg/auth/azure.go

```diff
@@ -138,15 +139,23 @@ func StreamInterceptor(
 	return handler(srv, stream)
 }
 
-func HttpAuthMiddleWare(resp http.ResponseWriter, req *http.Request, jwks *keyfunc.JWKS, clientId string, tenantId string) error {
+func HttpAuthMiddleWare(resp http.ResponseWriter, req *http.Request, jwks *keyfunc.JWKS, clientId string, tenantId string, allowedPaths []string, allowedPrefixes []string) error {
```

Comment:
please add a test for the paths/prefixes

Reply:
updated